controllers/azurejson_machinepool_controller_test.go

```diff
@@ -210,3 +217,140 @@ func TestAzureJSONPoolReconciler(t *testing.T) {
 		})
 	}
 }
+
+func TestAzureJSONPoolReconcilerUserAssignedIdentities(t *testing.T) {
+	g := NewWithT(t)
+	ctrlr := gomock.NewController(t)
+	defer ctrlr.Finish()
+	req := ctrl.Request{NamespacedName: types.NamespacedName{Name: "fake-machine-pool", Namespace: "fake-ns"}}
+	ctx := context.Background()
+	scheme := runtime.NewScheme()
```

Comment:
Similar to the other PR, I think you can use this existing function: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/controllers/azurejson_machine_controller_test.go#L240

Reply:
Utilized the `newScheme` test method to grab all the schemes as suggested.  Resolving.